jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -190,7 +190,7 @@ pipeline {
                     }
                 } // end of mvn verify stage
 
-                stage('Unit Test') {
+                stage('integration test') {
```

Comment:
Let's also rename the agent LABEL and WORKSPACE names from
premerge-test-it-${BUILD_TAG}
premerge-test-ut-${BUILD_TAG}
 "${CUSTOM_WORKSPACE}-it" 
 "${CUSTOM_WORKSPACE}-ut" 
to sth like
 "xxx-p1"
 "xxx-p2"
which does not have specific meanings in case we need to update scripts arguments again